disparse-core/src/test/java/disparse/discord/DispatchIntegrationTests.java

```diff
@@ -56,9 +57,9 @@ public static void fooBarBaz(TestDiscordRequest req, FooOpts opts) {
   }
 
   @CommandHandler(commandName = "cooldown")
-  @Cooldown(amount = 50, unit = ChronoUnit.MILLIS, sendCooldownMessage = true)
+  @Cooldown(amount = 50, unit = ChronoUnit.MILLIS, sentCooldownStrategy = CooldownStrategy.ENABLED)
   public static void cooldown(TestDiscordRequest req) {
-    req.getDispatcher().sendMessage(null, "test");
+    req.getDispatcher().reactToMessage(null);
```

Comment:
If you change this I think the test should not pass cause it looks for the "test" message explicitly.  Are the tests passing on this build?  It will be a bit before I am able to pull down the changes locally.  Other than that, looks good to me on all the changes!